README.md

```diff
@@ -193,6 +193,42 @@ dbt-invoke properties.delete <options>
   - `<options>` uses the same arguments as for creating/updating property
     files, except for `--threads`.
 
+
+### Applying default properties using `dbt_invoke_template.yml`
+
+You can pass custom default properties to be added to property files on update.
+To do so, add a `dbt_invoke_template.yml` at the root of your dbt project,
+using the following convention:
+
+
+```
+model:
```

Comment:
I might be nitpicking a bit here, but I think we should change `model`, `seed`, `snapshot`, and `analysis` to their plural versions; `models`, `seeds`, `snapshots`,  and `analyses` so that `dbt_invoke_template.yml` matches the convention used in property files. Thoughts?

Reply:
I agree with you! I think it's better to align to the used convention, I'll update `properties.py` to use plural versions instead.